.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Description of change
+
+Please write a summary of your changes and why you made them. Be sure to reference any related issues by adding `fixes # (issue)`.
+
+## Type of change
+
+Choose a type of change, and delete any options that are not relevant.
+
+- Bug fix (a non-breaking change which fixes an issue)
+- Enhancement (a non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- Documentation Fix
+
+## How the change has been tested
+
+Describe the tests that you ran to verify your changes.
+
+Make sure to provide instructions for the maintainer as well as any relevant configurations.
+
+## Change checklist
+
+Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.
+
+- [ ] My code follows the contribution guidelines for this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests using ginkgo that prove my fix is effective or that my feature works
```

Comment:
❔ ginkgo?

Reply:
Ginkgo should be removed. That's come over from the Go client library docs.